src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py

```diff
@@ -204,6 +208,11 @@ def __discoverCredentialsToUse(self):
            -> if KW_SKIP_CA_CHECK is not in kwargs and we are using the certificates,
                 set KW_SKIP_CA_CHECK to false in kwargs
            -> if KW_SKIP_CA_CHECK is not in kwargs and we are not using the certificate, check the skipCACheck
+        * Baerer token:
+          -> If KW_USE_ACCESS_TOKEN in kwargs, sets it in self.__useAccessToken
+          -> If not, check "/DIRAC/Security/UseTokens", and sets it in self.__useAccessToken
+              and kwargs[KW_USE_ACCESS_TOKEN]
+          -> If DIRAC_USE_ACCESS_TOKEN' in os.environ, sets it in self.__useAccessToken
```

Comment:
You should add this to this to *environment_variable_configuration.rst*

Reply:
added